recipe/run_test.bat

```diff
@@ -0,0 +1 @@
+%PYTHON% -c "import test_netcdf; test_netcdf.test_netcdf()"
```

Comment:
We don't need line 95 with this file, right?

Reply:
You are right, I think. I first only had the line in the meta.yaml, but that did not get run, so I tried to add a run_test.bat file, but that also did not get run, so I kept both and hoped to make at least one of them work, ready to clean up the other.

Reply:
I know the drill, testing on Windows without a local machine is... tiresome to say the least. Thanks for your efforts!